library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Actually, we want to move all non-native JS packages (including the version of bootstrap used in the TAGS system which is in /interface and any others we find floating around) _into_ the /assets directory as a migration...so when adding the function to other code, point all of it at the /assets directory.  library/js should ONLY be for NATIVE js stuff that is purely a LibreHealth maintenance item.  This is an ongoing cleanup effort.

In answer to your actual question, yes, awesome, sounds great, make a huge function unless you see a clear grouping that says "these will never be called with those others" sort of case.

Document in the header file how the file should be used and maintained, and any other useful things you come up with.  We want enough in-line documentation to make it clear what will be allowed, and what will not.  The file is for functions that organize and simplify the calling of includes, and for making full migrations or inclusions of new versions as simple as adding the new package and editing the headers.inc.php